src/engine/courses/Course.cpp

```diff
@@ -40,6 +45,204 @@ void ResizeMinimap(MinimapProps* minimap) {
     }
 }
 
+static std::unordered_set<std::string> gModifiedGfxSet;
+
+static void SwapTriangleVertices(Gfx* cmd) {
+    int8_t opcode = cmd->words.w0 >> 24;
+    
+    if (opcode == G_TRI1) {
+        uint32_t w1 = cmd->words.w1;
+        uint8_t v0 = (w1 >> 16) & 0xFF;
+        uint8_t v1 = (w1 >> 8) & 0xFF;
+        uint8_t v2 = w1 & 0xFF;
```

Comment:
These proccessing routines need to eventually go somewhere else.

We'll figure that out later.